app/containers/Organisations/UserOrganisation.js

```diff
@@ -26,6 +27,26 @@ export const UserOrganisationComponent = props => {
           {userOrganisation.status}
         </Badge>
       </td>
+      <td>
+        {' '}
+        {userOrganisation.status === 'active' ? (
+          <Link
+            href={{
+              pathname: '/user-organisation-facilities',
+              query: {
+                organisationId:
+                  userOrganisation.organisationByOrganisationId.id,
+                organisationName:
+                  userOrganisation.organisationByOrganisationId.operatorName
+              }
+            }}
+          >
+            <Button variant="primary">Go</Button>
+          </Link>
+        ) : (
+          <Button variant="danger">X</Button>
```

Comment:
Shoud the "Go" button be disabled (or not visible) instead of having a button that does nothing?